lib/test/iso/bmff/findBoxByType.test.ts

```diff
@@ -1,11 +1,11 @@
-import { assert, describe, findBoxByType, it } from './util/box.ts';
+import { assert, describe, findBoxByType, it, type MediaDataBox } from './util/box.ts';
 import { load } from './util/load.ts';
 
 describe('find a box by type', function () {
 	it('find a box by type', function () {
 		// Sample 'ftyp' box (20 bytes)
 		const buffer = load('./captions.mp4');
-		const box = findBoxByType(buffer, 'mdat');
+		const box = findBoxByType(buffer, 'mdat') as MediaDataBox;
```

Comment:
Ideally, based on `mdat`, it should infer `MediaDataBox | undefined`, to avoid type assertions

```suggestion
		const box = findBoxByType(buffer, 'mdat') as MediaDataBox;
```